recipes/safe-pysha3/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "safe-pysha3" %}
+{% set version = "1.0.4" %}
+{% set commit = "54be7e36b56c4734ca358c26b7734516cd748189" %}
+{% set sha256 = "292548b0e18d6043a468185d6ce27761204ebf4d694daef956b6ffb52a6871a8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/5afe/pysha3/archive/{{ commit }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: {{ PYTHON }} -m pip install . -vvv
+  number: 0
+  skip: true  # [py<=3.8]
```

Comment:
Invalid and unnecessary skip
```suggestion
```

Reply:
Hi @xhochy , This seems to be needed. The package cannot be used with python 3.8. How should I reflect this in the recipe, I tried several times, please advise

Reply:
Just omit this. We don't build packages for Python<3.8 on conda-forge anymore. As this is a non-noarch package, it will be build for each Python release individually.

Reply:
It does try to build for python 3.8 and fails:
```
Resolved dependencies, will be built in the following order:
    safe-pysha3-1.0.4-python_3.8.* *_cpython-on-linux-64
...
Modules/_sha3/sha3module.c:734:9: warning: implicit declaration of function 'Py_SET_TYPE'; did you mean 'Py_TYPE'? [-Wimplicit-function-declaration]
    734 |         Py_SET_TYPE(type, &PyType_Type); \
        |         ^~~~~~~~~~~
  Modules/_sha3/sha3module.c:744:5: note: in expansion of macro 'init_sha3type'
    744 |     init_sha3type("sha3_224", &SHA3_224type);
        |     ^~~~~~~~~~~~~
```
This is an issue in python 3.8